app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -304,11 +304,11 @@ export const RuleGroup = ({ pluginName, ruleDescription, rules, isPluginEnabled,
                 data={rules.map((rule) => ({
                   name: rule.ruleName,
                   ...rule,
-                  ruleFields,
                 }))}
                 actions={actions}
                 onToggle={onToggleHandler}
                 ruleItemContent={NotificationRuleContent}
+                ruleItemContentProps={ruleFields}
```

Comment:
Following the comment above it will be 
```suggestion
                ruleItemContentProps={{ ruleFields }}
```